src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -234,27 +275,49 @@ public Range(int min, int? max)
                 AutoEnd = max == null;
             }
 
+            /// <summary>
+            /// The minimum inclusive index of the column.
```

Comment:
>The minimum inclusive index of the column. [](start = 16, length = 42)

would phrasing it as follows sound better?

The minimum index of the column, inclusive.